### DISCUSSION TITLE
Space and Stream support Multi language on eXo

### DISCUSSION THREAD

**Discussion by Rajamuthukrishna** [exoplatform/exo-community#47] [Q&A]
Hello Team,

Can you please help to create space and stream for different languages.

If we can able to choose another language that language only display in space and stream.

**Comment by Rajamuthukrishna** [comment]
@rdenarie @plamarque @boubaker @ahamdi  - Can you please confirm, We can able to use multi language in space and stream.

**Comment by rdenarie** [comment]
Hello @Rajamuthukrishna 

The multi language is defined by user : each user will see the platform in the same language of his browser, or can configure it in personnal settings.

So, ALL the interface will use the choosen language, in all spaces. 

For user generated content (activities, comments, notes ...), the user will see it in the language the author use to create it.
In enterprise version, we have solution to automatically translate user generated content with integration of Google Translate or DeepL.
In the next version (6.5), which should be released at the end of the year, we will have solutions to translate notes, and news, differents language the user can choose.

Does it answer your question ?

**Reply by Rajamuthukrishna** [reply]
yes @rdenarie .Thanks for the prompt response.
